app/views/stories/tagged_articles/_sidebar.html.erb

```diff
@@ -2,28 +2,32 @@
   <div id="sidebar-wrapper-left" class="sidebar-wrapper sidebar-wrapper-left">
     <div class="sidebar-bg" id="sidebar-bg-left"></div>
     <aside class="side-bar">
-      <% if @tag && @tag.rules_html.present? %>
+      <% if @tag %>
         <%= application_policy_content_tag("div", record: Article, query: :create?, class: "widget") do %>
-          <header>
-            <h4><%= t("views.tags.sidebar.guidelines") %></h4>
-          </header>
-          <div class="widget-body">
-            <%= sanitize @tag.rules_html %>
-            <a class="crayons-btn crayons-btn--s" href="/new/<%= @tag.name %>">
-              <%= t("views.tags.sidebar.new") %>
-            </a>
+          <a class="crayons-btn crayons-btn--s" href="/new/<%= @tag.name %>">
```

Comment:
My read on this change is that the link will now be above the header regardless of whether we have guidelines or not.  I think this is okay, but it definitely changes some potentially expected layout.

Reply:
I'd love @ludwiczakpawel or @anujbhavsar96 to weigh in; from a "logic" standpoint things work out.